app/services/resource_recommendation.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ResourceRecommendation
+  def initialize(moment)
+    @moment = moment
+    @moment_keywords = []
+  end
+
+  def resources
```

Comment:
Similar consideration for this class—I wonder if it could be worth considering something like:

```ruby
ResourceRecommendations.for(moment)

# or

ResourceRecommendations.new(moment).call
```